text/machine_comprehension/gpt-2/GPT2-export.py

```diff
@@ -1,5 +1,6 @@
 import torch
```

Comment:
Can we make the updates to the script in ShuffleNet_v2 model as well?
https://github.com/onnx/models/blob/master/vision/classification/shufflenet_v2/shufflenet-export.py

Reply:
Sure. Done.